models/Document/Editable/Date.php

```diff
@@ -73,7 +74,7 @@ public function getDataEditmode() /** : mixed */
     public function frontend()
     {
         if ($this->date instanceof \DateTimeInterface) {
```

Comment:
But I don't see a case that the date can be set an DateTimeInterface without Carbon. I think it is save to change it directly here:
```suggestion
        if ($this->date instanceof Carbon) {
```

Reply:
mhh, that's right, everything was and will(in 11) be `Carbon`  anyways